### ISSUE TITLE
InPlaceABN and InPlaceABNSync do not generate the same results as torch.nn.BatchNorm2d

### ISSUE THREAD

**Issue by denru01** [mapillary/inplace_abn#114]
I noticed that InPlaceABN and InPlaceABNSync do not generate the same results as torch.nn.BatchNorm2d and manual computation with the same parameters. I tested ABN, InPlaceABN and InPlaceABNSync, and only ABN gave the matched answer. Please refer to the following code. I am using Pytorch 1.0 and the head of the master branch. Did I do something wrong?

Code:
```
import torch
from modules import InPlaceABNSync

running_mean = 13.97
running_var = 313.7588
weight = -0.0059
bias = 0.1408
input_value = -9.3330
eps = 1e-05

class BatchNorm2DNetwork(torch.nn.Module):
    def __init__(self):
        super(BatchNorm2DNetwork, self).__init__()
        self.bn = torch.nn.BatchNorm2d(1, eps=eps)
        self.bn.running_mean = torch.tensor([running_mean])
        self.bn.running_var = torch.tensor([running_var])
        self.bn.weight = torch.nn.Parameter(torch.tensor([weight]))
        self.bn.bias = torch.nn.Parameter(torch.tensor([bias]))

    def forward(self, input):
        output = self.bn(input)
        return output

class InplaceABNNetwork(torch.nn.Module):
    def __init__(self):
        super(InplaceABNNetwork, self).__init__()
        self.bn = InPlaceABNSync(1, activation='none', eps=eps)
        self.bn.running_mean = torch.tensor([running_mean])
        self.bn.running_var = torch.tensor([running_var])
        self.bn.weight = torch.nn.Parameter(torch.tensor([weight]))
        self.bn.bias = torch.nn.Parameter(torch.tensor([bias]))

    def forward(self, input):
        output = self.bn(input)
        return output

def ManualBatchNorm(input):
    output = ((input-running_mean)/((running_var+eps)**0.5))*weight+bias
    return output

net_bn2d = BatchNorm2DNetwork().eval()
net_inplace = InplaceABNNetwork().eval()

input_bn2d = torch.ones((1,1,1,1))*input_value
input_inplace = torch.ones((1,1,1,1))*input_value

print('net_bn2d: {}'.format(net_bn2d(input_bn2d)))
print('net_inplace: {}'.format(net_inplace(input_inplace)))
print('manual: {}'.format(ManualBatchNorm(input_value)))
```

Result:
```
net_bn2d: tensor([[[[0.1486]]]], grad_fn=<NativeBatchNormBackward>)
net_inplace: tensor([[[[0.1330]]]], grad_fn=<InPlaceABNSyncBackward>)
manual: 0.14856186137771327
```

**Comment by denru01** [comment]
I think this is a duplicated issue as #92, so I close it.
However, I would suggest putting this note into the readme because this behavior is highly not expected and took me a whole night debugging the code.
Thanks!

**Comment by rotabulo** [comment]
@denru01 I have added a paragraph in the README that should clarify how we implement the scaling factor. Le me know if you think it's still not clear.

**Comment by denru01** [comment]
@rotabulo That is awesome. Thanks!
